### ISSUE TITLE
VETO Explain

### ISSUE THREAD

**Issue by abdulsami34** [splewis/get5#327]
Hi there,
I'm currently confused in veto.  Can you please explain the method of veto

Can you please explain in bo1 like we will cut every map and the last two remaining will be vetoed as well or will be picked to play? like there are two maps left eg

de_mirage
de_cache

We want to play mirage for example. Should we veto cache to play mirage or pick it like in warmod? because i haven't tested out the veto system yet because you know the confusion. the other team says just do it in chat.

**Comment by MikkelDK** [comment]
Hi,

You can read here: [Map VETO](https://github.com/splewis/get5/wiki/Map-Vetoes)

BO1:
Each team alternatives vetoing, vetoing 3 maps each. The 7th map left will be played. A knife round will be used to decide what the starting sides are.

tldr: ban/ban/ban/ban/ban/ban/last map is played

**Comment by splewis** [comment]
I think the wiki and post here explain it.

The veto text menu should explicitly say you are picking a map to veto, or picking a map to play.

It's not a bad idea to print some text in the text chat to make it clear again though. If I was a player (without  knowing how get5 worked) I'd second guess my choice at the last veto stage.
